stripe/src/main/java/com/stripe/android/view/CardMultilineWidget.java

```diff
@@ -445,13 +454,6 @@ private void updateDrawable(
             @DrawableRes int iconResourceId,
             boolean needsTint) {
 
-        // Don't update the drawable more than is necessary.
```

Comment:
We don't update that often, and this caused a bug in SDK < 21 where the tint wasn't applied to the default card icon.